pkg/kudoctl/cmd/init/manager.go

```diff
@@ -57,18 +57,22 @@ func NewOptions(v string) Options {
 }
 
 // Install uses Kubernetes client to install KUDO.
-func Install(client *kube.Client, opts Options) error {
-	clog.V(4).Printf("installing prereqs")
-	if err := installPrereqs(client.KubeClient, opts); err != nil {
-		return err
+func Install(client *kube.Client, opts Options, crdOnly bool) error {
+	if !crdOnly {
```

Comment:
actually, now that this is here, is there any reason to separate prereqs from manager? after a quick look through the code, I _think_ you could just delete this whole block and put installPrereqs inside of the block on line 71, but lmk if I'm missing anything.

Reply:
(I don't think the prereqs are needed to install the CRDs, either)

Reply:
without a doubt we don't need prereq for crd... moving them would clean up the code a little... thanks!